packages/renderer/src/lib/dashboard/NotificationCardItem.svelte

```diff
@@ -22,12 +22,12 @@ async function removeNotification(id: number) {
     </div>
     <div class="flex flex-col text-sm space-y-2">
       <div class="font-bold" aria-label="Notification title">{notification.title}</div>
-      <div aria-label="Notification description"><Markdown>{notification.body ?? ''}</Markdown></div>
+      <div aria-label="Notification description"><Markdown markdown="{notification.body ?? ''}" /></div>
```

Comment:
Why does the markdown is changed ?

Reply:
it looks like with svelte5 I had at some point the content not being displayed using the other syntax but it was also not 100%